app/src/main/java/org/mifos/mobile/ui/activities/base/BaseActivity.java

```diff
@@ -214,15 +215,29 @@ public void replaceFragment(Fragment fragment, boolean addToBackStack, int conta
 
         if (!fragmentPopped && getSupportFragmentManager().findFragmentByTag(backStateName) ==
                 null) {
+            Fragment previousFragment = getSupportFragmentManager().findFragmentById(containerId);
             FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
+            transition(previousFragment, fragment);
             transaction.replace(containerId, fragment, backStateName);
             if (addToBackStack) {
                 transaction.addToBackStack(backStateName);
             }
-            transaction.commit();
+            transaction.commitAllowingStateLoss();
         }
     }
 
+    public void transition(Fragment previousFragment, Fragment nextFragment){
+        if(previousFragment!=null) {
```

Comment:
After `if` and `!=` one white space is needed that is why your build is failing. You can simply solve it by formating the code by pressing `ctrl+alt+l` :)